### ISSUE TITLE
Adding Dummy Classifier and regressor as Baseline to compare_models

### ISSUE THREAD

**Issue by reza1615** [pycaret/pycaret#1601]
Sometimes we need a baseline model which acts like common sense or human.
Would you please add the Sklearn [Dummy ](https://scikit-learn.org/stable/modules/generated/sklearn.dummy.DummyClassifier.html#sklearn.dummy.DummyClassifier) Classifier and regressors to the compare_models with different strategies 

**Regressor** 
* “mean”: always predicts the mean of the training set
* “median”: always predicts the median of the training set
* “quantile”: always predicts a specified quantile of the training set, provided with the quantile parameter.
* “constant”: always predicts a constant value that is provided by the user.

**Classifier**
* “stratified”: generates predictions by respecting the training set’s class distribution.
* “most_frequent”: always predicts the most frequent label in the training set.
* “prior”: always predicts the class that maximizes the class prior (like “most_frequent”) and predict_proba returns the class prior.
* “uniform”: generates predictions uniformly at random.
* “constant”: always predicts a constant label that is provided by the user. This is useful for metrics that evaluate a non-majority class

This feature can be activated by a variable like `add_Dummy_model= True`

`compare_models(add_Dummy_model= True)`

**Comment by reza1615** [comment]
**Added Dummy Classifier**

![Screenshot from 2021-09-23 09-38-34](https://user-images.githubusercontent.com/2202483/134517721-a5624232-3489-47a5-b8e8-3f1a5a871eb6.png)

**Added Dummy Regressor**
![Screenshot from 2021-09-23 09-43-20](https://user-images.githubusercontent.com/2202483/134518166-fe5a0b3d-3551-4d10-ba68-0db4df647267.png)
